packages/cacti-copm-core/Makefile

```diff
@@ -0,0 +1,194 @@
+WORKSPACE ?= $(shell realpath `pwd`/../../)
```

Comment:
I'd prefer it be moved to cacti-copm-test then. Let's completely separate the testing setup and logic from the generic components.

Ideally, I'd separate the test setups from the testing logic too, so that the testing logic could get a "spec" for the test setup and run generic tests on the network it's been given, but I think the Cacti package structure doesn't make this easy.

Moving this logic to the cacti-copm-test package seems to be the low-hanging-fruit option now, so let's do that. Perhaps later, we can refactor this to move the testnet launch logic to `examples` (as @sandeepnRES suggests), thereby making cacti-copm-test also oblivious to the network structure.

Reply:
I have moved all the Makefiles to cacti-copm-test.  The Makefile_corda and Makefile_fabric files have 3 key targes: setup,  lock-network, and pledge-network.  These can be referenced either directly:
 
`make -f Makefile_corda setup`

or via wildcard targets in the main makefile:

`make corda-setup`

Future network types can follow this pattern.